docs/UserGuide.md

```diff
@@ -245,7 +240,6 @@ The location for this file is `./data/tbmManager.json` (indicated at the bottom
       "tags" : [ "romantic", "holidayPlans", "travel" ]
     } ]
   } ]
-  . . . 
```

Comment:
Should this be left inside? Since it's supposed to be a snippet. Or maybe change the desc to "Here's a sample showing what this editable text file will look like:"

Reply:
oh yea, I thought it was mistakenly left in since intellij is being annoying